spec.md

```diff
@@ -727,6 +727,9 @@ Plugins MAY create 3 types of volumes:
 - From an existing snapshot. When plugin supports `CREATE_DELETE_VOLUME` and `CREATE_DELETE_SNAPSHOT` OPTIONAL capabilities.
 - From an existing volume. When plugin supports cloning, and reports the OPTIONAL capabilities `CREATE_DELETE_VOLUME` and `CLONE_VOLUME`.
 
+If CO requests a volume to be created from existing snapshot or volume and the requested size of the volume is larger than the original snapshotted (or cloned volume), the Plugin can either refuse such a call with `OUT_OF_RANGE` error or MUST provide a volume that, when presented to a workload by `NodePublish` call, has both the requested (larger) size and contains data from the snapshot (or original volume).
```

Comment:
what if the restored file system not supported for resize at this moment, should Plugin return the same error code?

Reply:
already anwered by @jdef below, yes, OUT_OF_RANGE is IMO appropriate in this case too.